README.md

```diff
@@ -9,8 +9,23 @@ Each of these Terraform projects are to be layered on top of each other, running
 
 ### [0. bootstrap](./0-bootstrap/)
 
-This stage executes the [CFT Bootstrap module](https://github.com/terraform-google-modules/terraform-google-bootstrap) which bootstraps an existing GCP organization, creating all the required GCP resources & permissions to start using the Cloud Foundation Toolkit (CFT).
-This includes; projects, service accounts and a Terraform state bucket. After executing this step, you will have the following structure:
+This stage executes the [CFT Bootstrap module](https://github.com/terraform-google-modules/terraform-google-bootstrap) which bootstraps an existing GCP organization, creating all the required GCP resources & permissions to start using the Cloud Foundation Toolkit (CFT). This includes:
```

Comment:
We should have some indication that cloud build is default and to uncomment if Jenkins needs to be enabled.

Reply:
Yeah I think this makes sense.

Reply:
I have all steps to follow to deploy Bootstrap with the Jenkins-Agent module. Will share with you now.